### DISCUSSION TITLE
Question about Csv GwGm files

### DISCUSSION THREAD

**Discussion by aeomath** [OpenMDAO/Aviary#286] [Q&A] [answered]
Hello , 

I am a new user of aviary and while running some mission using GASP models I've noticed in the input file some inputs which are not used by aviary during the analysis. 

INGASP.KNAC,1
INGASP.KODEAC,7
INGASP.KODECL,7
INGASP.KODETO,5
INGASP.KODETR,6
---- --------------
INGASP.HBTP,0.333
INGASP.HIR,0.015
INGASP.HNCRU,37500
INGASP.HPORT,0
INGASP.HRI,2500
INGASP.HSCREQ,20000

I have no idea what these acronyms stand for. If someone could guide me and give some doc about this I would love to 

Thank you for you help :)

**Comment by ehariton** [comment]
GASP source docs would detail what these are:
https://ntrs.nasa.gov/api/citations/19810010562/downloads/19810010562.pdf

It looks like something like HNCRU is also referenced as CRALT, the mission cruise altitude in ft.

**Comment by jkirk5** [comment, marked as answer]
For some additional context, the GASP-based models that we include with Aviary were converted from an actual GASP input file to a csv using Aviary's `fortran_to_aviary` utility function. That util saves a list of all variables that Aviary didn't recognize, which can happen for two reasons: 1) Aviary simply does not use/support that variable, or 2) the notion of that variable does exist in the code, but there isn't a simple 1:1 name match, which often happens for mission-related parameters. (or 3, which is it accidentally got skipped due to a bug!)

Right now the util isn't communicating _why_ a variable isn't converted, so a little manual checking of that list is needed. Unfortunately that does mean that familiarity with the legacy tool is currently required for successfully converting an input file.
